GiphySearch/GifCell.swift

```diff
@@ -6,34 +6,44 @@
 //
 
 import UIKit
+
 class GifCell: UICollectionViewCell {
+    
+    @IBOutlet private var loader: UIActivityIndicatorView!
+    
     var gif: Gif? {
         didSet {
             fetchGif()
         }
     }
     
-    @IBOutlet weak var loader: UIActivityIndicatorView!
-    var gifView: UIImageView = {
-        let v = UIImageView()
-        v.contentMode = .scaleAspectFit
-        return v
+   private var gifView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        
+        return imageView
     }()
     
+    override func prepareForReuse() {
+        super.prepareForReuse()
```

Comment:
When overriding make sure to call super, even thought default `prepareForReuse` does nothing, Apple engineers in documentation of method suggest to call super eitherway

> The default implementation of this method does nothing. However, when overriding this method, it is recommended that you call super anyway.